gateway-ha/src/main/java/io/trino/gateway/ha/module/ClusterStateListenerModule.java

```diff
@@ -52,8 +56,14 @@ public List<TrinoClusterStatsObserver> getClusterStatsObservers(
             BackendStateManager backendStateManager)
     {
         observers = new ArrayList<>();
+        NotifierConfiguration notifierConfiguration = getConfiguration().getNotifier();
+        Notifier emailNotifier = new EmailNotifier(notifierConfiguration);
```

Comment:
Please inline emailNotifier variable. No need to initialize when the notification is disabled.

Reply:
done